.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+indent_style = space
```

Comment:
@nlok5923 Are you sure `space` would be better than `tab`  ?

Reply:
@Rahulm2310  i ventured through some of the React project and find space as good formatting standard.